diff -urNad mysql-5.5.43-orig/vio/viossl.c mysql-5.5.43/vio/viossl.c
--- mysql-5.5.43-orig/vio/viossl.c	2015-02-27 10:30:34.000000000 +0200
+++ mysql-5.5.43/vio/viossl.c	2015-04-16 19:21:20.000000000 +0300
@@ -172,8 +172,10 @@
   SSL_SESSION_set_timeout(SSL_get_session(ssl), timeout);
   SSL_set_fd(ssl, vio->sd);
 #ifndef HAVE_YASSL
+#ifdef SSL_OP_NO_COMPRESSION
   SSL_set_options(ssl, SSL_OP_NO_COMPRESSION);
 #endif
+#endif
 
   if ((r= connect_accept_func(ssl)) < 1)
   {
