diff -urNad mysql-5.5.39-orig/vio/viossl.c mysql-5.5.39/vio/viossl.c
--- mysql-5.5.39-orig/vio/viossl.c	2014-07-19 09:25:07.000000000 +0300
+++ mysql-5.5.39/vio/viossl.c	2014-09-06 11:02:36.000000000 +0300
@@ -172,8 +172,10 @@
   SSL_SESSION_set_timeout(SSL_get_session(ssl), timeout);
   SSL_set_fd(ssl, vio->sd);
 #ifndef HAVE_YASSL
+#ifdef SSL_OP_NO_COMPRESSION
   SSL_set_options(ssl, SSL_OP_NO_COMPRESSION);
 #endif
+#endif
 
   if ((r= connect_accept_func(ssl)) < 1)
   {
