# Fix system zlib detection and version check
diff -urNad mysql-8.0.33/cmake/zlib.cmake mysql-8.0.33-new/cmake/zlib.cmake
--- mysql-8.0.33/cmake/zlib.cmake	2023-03-16 19:22:37.000000000 +0200
+++ mysql-8.0.33-new/cmake/zlib.cmake	2023-04-19 14:40:03.087736273 +0300
@@ -45,6 +45,7 @@
       ${version_part} "${${version_part}}")
   ENDFOREACH()
   SET(ZLIB_VERSION "${ZLIB_VER_MAJOR}.${ZLIB_VER_MINOR}.${ZLIB_VER_REVISION}")
+  SET(ZLIB_VERSION "${ZLIB_VERSION}" PARENT_SCOPE)
   SET(ZLIB_VERSION "${ZLIB_VERSION}" CACHE INTERNAL "ZLIB major.minor.step")
   MESSAGE(STATUS "ZLIB_VERSION (${WITH_ZLIB}) is ${ZLIB_VERSION}")
   MESSAGE(STATUS "ZLIB_INCLUDE_DIR ${ZLIB_INCLUDE_DIR}")
@@ -60,6 +61,9 @@
       TARGET_INCLUDE_DIRECTORIES(zlib_interface SYSTEM INTERFACE
         ${ZLIB_INCLUDE_DIR})
     ENDIF()
+    SET(ZLIB_FOUND ${ZLIB_FOUND} PARENT_SCOPE)
+    FIND_ZLIB_VERSION(${ZLIB_INCLUDE_DIR})
+    SET(ZLIB_VERSION ${ZLIB_VERSION} PARENT_SCOPE)
   ENDIF()
 ENDFUNCTION(FIND_SYSTEM_ZLIB)
 
